optuna/visualization/_pareto_front.py

```diff
@@ -238,47 +239,56 @@ def _get_pareto_front_info(
     targets: Callable[[FrozenTrial], Sequence[float]] | None = None,
 ) -> _ParetoFrontInfo:
     if axis_order is not None:
-        warnings.warn(
-            "`axis_order` has been deprecated in v3.0.0. "
-            "This feature will be removed in v5.0.0. "
-            "See https://github.com/optuna/optuna/releases/tag/v3.0.0.",
-            FutureWarning,
+        msg = _deprecated._DEPRECATION_WARNING_TEMPLATE.format(
+            name="`axis_order`",
+            d_ver="3.0.0",
+            r_ver="5.0.0",
```

Comment:
It is very picky but we can remove line breaks by this:
```suggestion
            name="`axis_order`", d_ver="3.0.0", r_ver="5.0.0"
```